src/app.ts

```diff
@@ -14,6 +15,13 @@ import { notFoundHandler, errorHandler } from "./middleware";
 const createApp = () => {
   const app = express();
 
+  // This allows the query parser to parse up to 100 coordinates without adding indices.
+  // Anything over 100 would error out because indices are added. See CMR-10296 and
+  // https://github.com/ljharb/qs for more details.
+  app.set("query parser", function (str: string) {
+    return qs.parse(str, { arrayLimit: 10000 });
```

Comment:
Array limit is 10,000 but, comment is 100?

Reply:
Sorry about that. Was testing 10000.